org-code-javabuilder/protocol/src/main/java/org/code/protocol/SystemExitHelper.java

```diff
@@ -0,0 +1,10 @@
+package org.code.protocol;
```

Comment:
Should this be in the org.code.javabuilder package? It seems unlikely that we'd use it in the student-facing APIs.

Reply:
Oh yep good point, I'll move it